### DISCUSSION TITLE
Any other fine-tuned version of whisper?

### DISCUSSION THREAD

**Discussion by MadhuKiran1999** [openai/whisper#2414] [Q&A] [answered]
Is there any other better fine-tuned version of whisper models on Indic languages?

**Comment by samarasimhapeyala** [comment, marked as answer]
Yes, the Vasista fine-tuned models are indeed better suited for Indic languages. You can find them on Hugging Face at https://huggingface.co/vasista22. But these are fine tuned on whisper large v2 only

**Comment by Hu-nie** [comment]
@samarasimhapeyala  

Hello! I have a question.
Is there a way to apply a fine-tuned model to Whisper?
